service/kbupd/src/bin/kbupctl.rs

```diff
@@ -54,11 +56,7 @@ fn main() -> Result<(), failure::Error> {
 
     match subcommand_name {
         "info" | "status" => {
-            let print_fun = match subcommand_name {
-                "info" => print_info,
-                "status" => print_status,
-                _ => unreachable!(),
-            };
+            let subcommand_name = subcommand_name.to_string();
```

Comment:
IIUC this is unnecessary, especially since we check `subcommand_name.as_str()` below